Sphinx docs for sympy.liealgebras

As of now, docs for Lie Algebra Module is not there.

Hello @aktech and @moorepants I want to fix this issue, So to generate the html from the doc I am simply using 

$ make html

but I am getting the following error while the execution of the third process
preparing documents... 
Theme error:
unsupported theme option 'visitedlinkcolor' given
make: *** [html] Error 1

I am working on Sympy using the virtualenv wrapper and installed sphinx in virtualenv as well. Please let me know the problem.

--- a/doc/src/modules/index.rst
+++ b/doc/src/modules/index.rst
@@ -27,6 +27,7 @@ access any SymPy module, or use this contens:
    galgebra/index.rst
    geometry/index.rst
    integrals/integrals.rst
+   liealgebras/index.rst
    logic.rst
    matrices/index.rst
    polys/index.rst

--- a/doc/src/modules/liealgebras/index.rst
+++ b/doc/src/modules/liealgebras/index.rst
@@ -1,6 +1,6 @@
-========
+==================
 Lie Algebra Module
-========
+==================
 
 .. automodule:: sympy.liealgebras
 
@@ -40,4 +40,3 @@ Lie Algebra Module
 .. autofunction:: sympy.liealgebras.dynkin_diagram.DynkinDiagram
 
 .. autofunction:: sympy.liealgebras.cartan_matrix.CartanMatrix
-

--- a/doc/src/modules/liealgebras/index.rst
+++ b/doc/src/modules/liealgebras/index.rst
@@ -1,0 +1,43 @@
+========
+Lie Algebra Module
+========
+
+.. automodule:: sympy.liealgebras
+
+.. autoclass:: sympy.liealgebras.root_system.RootSystem
+    :members:
+
+.. autoclass:: sympy.liealgebras.type_a.TypeA
+    :members:
+
+.. autoclass:: sympy.liealgebras.type_b.TypeB
+    :members:
+
+.. autoclass:: sympy.liealgebras.type_c.TypeC
+    :members:
+
+.. autoclass:: sympy.liealgebras.type_d.TypeD
+    :members:
+
+.. autoclass:: sympy.liealgebras.type_e.TypeE
+    :members:
+
+.. autoclass:: sympy.liealgebras.type_f.TypeF
+    :members:
+
+.. autoclass:: sympy.liealgebras.type_g.TypeG
+    :members:
+
+.. autoclass:: sympy.liealgebras.weyl_group.WeylGroup
+    :members:
+
+.. autoclass:: sympy.liealgebras.cartan_type.CartanType_generator
+    :members:
+
+.. autoclass:: sympy.liealgebras.cartan_type.Standard_Cartan
+    :members:
+
+.. autofunction:: sympy.liealgebras.dynkin_diagram.DynkinDiagram
+
+.. autofunction:: sympy.liealgebras.cartan_matrix.CartanMatrix
+

